app/grandchallenge/cases/models.py

```diff
@@ -325,6 +326,32 @@ def image_file_path(instance, filename):
     )
 
 
+class DICOMImageSet(UUIDModel):
+    image_set_id = models.CharField(
+        max_length=32,
+        unique=True,
+        help_text="The ID of the image set in AWS Health Imaging.",
+    )
+    dicom_image_set_upload = models.OneToOneField(
+        to="DICOMImageSetUpload",
+        editable=False,
+        null=True,
+        on_delete=models.SET_NULL,
```

Comment:
Do we want this to be nullable? I think that we always would want information about the origin of the image set, so maybe keep it and protect it on deletion?